featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -146,7 +190,17 @@ def nunique(x):
 
 
 class NumTrue(AggregationPrimitive):
-    """Finds the number of 'True' values in a boolean."""
+    """Calculates the number of `True` values.
```

Comment:
maybe say "Counts the number of `True` values."?

Reply:
Fixed.